### PR TITLE
Feature/3.0/siteselector implementation

### PR DESCRIPTION
**Pull request by sebastianthulin**

### CODE REVIEW

File: library/Customizer/Applicators/ComponentData.php [hunk 1]
```diff
@@ -41,8 +49,15 @@ public function applyComponentData()
                 add_filter('ComponentLibrary/Component/Data', function ($data) use ($filter) {
                     $contexts = is_string($data['context']) ? [$data['context']] : $data['context'];
 
+                    if(!in_array('component.siteselector', $contexts)) {
+                        return $data;
+                    }
```
Comment (thorbrink) [inline library/Customizer/Applicators/ComponentData.php:54]: Is this supposed to be here?

Comment (sebastianthulin) [reply library/Customizer/Applicators/ComponentData.php:54]: Nope 👯

Comment (sebastianthulin) [inline library/Customizer/Applicators/ComponentData.php:54]:
```suggestion
                    
```

File: library/Customizer/Applicators/ComponentData.php [hunk 2]
```diff
@@ -41,8 +49,15 @@ public function applyComponentData()
                 add_filter('ComponentLibrary/Component/Data', function ($data) use ($filter) {
                     $contexts = is_string($data['context']) ? [$data['context']] : $data['context'];
 
+                    if(!in_array('component.siteselector', $contexts)) {
```
Comment (sebastianthulin) [inline library/Customizer/Applicators/ComponentData.php:52]:
```suggestion
```

**Review by thorbrink — APPROVED** [review]
🥇

**Pull request merged**